nifi-toolkit/nifi-toolkit-tls/src/main/java/org/apache/nifi/toolkit/tls/TlsToolkitMain.java

```diff
@@ -86,6 +86,8 @@ protected Method getMain(String service) {
             printUsageAndExit("Unknown service: " + service, ExitCode.INVALID_ARGS);
         }
 
+        System.out.println("The TLS Toolkit is deprecated and targeted for removal in Apache NiFi 2.0.");
```

Comment:
Can/should we use the Deprecation Logger here?

And consider adding the `@Deprecated` annotation for `public` classes/methods referenced elsewhere so people get more warnings about this utility being removed in the future. The simple `System.out` could be easily missed (but I've not checked whether the standard NiFi Deprecation Logger would be available for us easily here)